libr/anal/sign.c

```diff
@@ -2192,6 +2226,37 @@ static int sig_hash_cmp(RSignHash *a, RSignHash *b) {
 	return strcmp (a->bbhash, b->bbhash);
 }
 
+static int sig_var_diff(RList *la, RList *lb) {
+	if (!la) {
+		return lb? -1: 0;
+	}
+	if (!lb) {
+		return 1;
+	}
+	int len = r_list_length (la);
+	int dif = len - r_list_length (lb);
+	if (dif) {
+		return dif;
+	}
+	size_t i;
+	for (i = 0; i < len; i++) {
+		const RAnalVarProt *a = r_list_get_n (la, i);
+		const RAnalVarProt *b = r_list_get_n (lb, i);
+		dif = a->delta - b->delta;
```

Comment:
can a or b be null :?

Reply:
I don't think so, but better to check then crash. I will update it to warn in that case too.